axelrod/tournament.py

```diff
@@ -3,32 +3,64 @@
 import logging
 import multiprocessing
 
-from .game import *
-from .result_set import *
-from .round_robin import *
+from .game import Game
+from .result_set import ResultSet
+from .tournament_type import round_robin
+from .payoff import payoff_matrix
+from .cooperation import cooperation_matrix
 
 
 class Tournament(object):
     game = Game()
 
-    def __init__(self, players, name='axelrod', game=None, turns=200,
+    def __init__(self, players, tournament_type=None, name='axelrod',
```

Comment:
Could the default just be `round_robin` this would be more readable? (Based on what is happening at line 48)